Common subdirectories: WebKit.orig/Source/WebCore/platform/webos/demoCA and WebKit/Source/WebCore/platform/webos/demoCA
diff -u WebKit.orig/Source/WebCore/platform/webos/LunaServiceMgr.cpp WebKit/Source/WebCore/platform/webos/LunaServiceMgr.cpp
--- WebKit.orig/Source/WebCore/platform/webos/LunaServiceMgr.cpp        2012-07-17 16:53:27.000000000 -0700
+++ WebKit/Source/WebCore/platform/webos/LunaServiceMgr.cpp     2012-07-17 21:48:30.000000000 -0700
@@ -267,7 +267,6 @@
  */
 void LunaServiceManager::cancel(LunaServiceManagerListener* inListener)
 {
-    bool retVal;
     LSError lserror;
 
     if (!inListener || !inListener->listenerToken)
diff -u WebKit.orig/Source/WebCore/platform/webos/PalmServiceBridge.cpp WebKit/Source/WebCore/platform/webos/PalmServiceBridge.cpp
--- WebKit.orig/Source/WebCore/platform/webos/PalmServiceBridge.cpp     2012-07-17 16:54:06.000000000 -0700
+++ WebKit/Source/WebCore/platform/webos/PalmServiceBridge.cpp  2012-07-17 21:49:01.000000000 -0700
@@ -37,7 +37,7 @@
 typedef std::set<PalmServiceBridge*> ServicesSet;
 typedef std::map<Document*, ServicesSet*> ServicesSetMap;
 
-static bool sDebugServices = (getenv("DEBUG_SERVICES") ? true : false);
+//static bool sDebugServices = (getenv("DEBUG_SERVICES") ? true : false);
 
 #ifndef NDEBUG
 static WTF::RefCountedLeakCounter serviceBridgeCounter("PalmServiceBridge");
@@ -239,7 +239,6 @@
 
 int PalmServiceBridge::call(const String& uri, const String& payload, ExceptionCode& ec)
 {
-    const char* callerId = 0;
     bool usePrivateBus = false;
 
     JSValue identifier;
diff -u WebKit.orig/Source/WebCore/platform/webos/PalmServiceBridge.h WebKit/Source/WebCore/platform/webos/PalmServiceBridge.h
--- WebKit.orig/Source/WebCore/platform/webos/PalmServiceBridge.h       2012-07-17 16:56:02.000000000 -0700
+++ WebKit/Source/WebCore/platform/webos/PalmServiceBridge.h    2012-07-17 16:56:12.000000000 -0700
@@ -1,7 +1,7 @@
 #ifndef PalmServiceBridge_h
 #define PalmServiceBridge_h
 
-#ifdef ENABLE(PALM_SERVICE_BRIDGE)
+#if ENABLE(PALM_SERVICE_BRIDGE)
 
 #include "ActiveDOMObject.h"
 #include "Event.h"

